OMPython/__init__.py

```diff
@@ -1773,6 +1773,19 @@ def getLinearMatrixValues(self,matrix):
             return tmpmatrix
         else:
             return np.zeros((0,0))
+        
+    def setCurrDir(mo, newPath):
+        path = newPath.replace('\\', '/')
+
+        if path[-1] == '/':
+            path = path[:-1]
+
+        if not os.path.exists(path):
+            raise Exception("Error: path does not exist")
+
+        mo.currDir = path
+        mo.requestApi('cd', entity=path)
```

Comment:
Check if the returned string of `cd` is same as `path` if yes then the call is successful. There is no need to check for error via `getErrorString`